pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -294,13 +325,34 @@ func (r *hibernationReconciler) Reconcile(ctx context.Context, request reconcile
 			// Allow hibernation (do not set condition) if hibernateAfterSyncSetsNotApplied duration has passed since cluster
 			// installed and syncsets still not applied
 			if cd.Status.InstalledTimestamp != nil && time.Now().Sub(cd.Status.InstalledTimestamp.Time) < hibernateAfterSyncSetsNotApplied {
-				return r.setHibernatingCondition(cd, hivev1.SyncSetsNotAppliedReason, "Cluster SyncSets have not been applied", corev1.ConditionFalse, cdLog)
+				changed := r.setHibernatingCondition(cd, hivev1.SyncSetsNotAppliedReason, "Cluster SyncSets have not been applied", corev1.ConditionFalse, cdLog)
+				rChanged := r.setReadyCondition(cd, hivev1.RunningReadyReason, clusterRunningMsg, corev1.ConditionTrue, cdLog)
```

Comment:
Here's a good example: knowing there's an installed timestamp and syncsets haven't been applied does _not_ mean the cluster is running. IMO we should either not touch the Ready condition at all; or change it to Unknown with the same reason/msg as the hibernating condition above.